lib/grpc-lwt/client.ml

```diff
@@ -18,13 +18,12 @@ let default_headers =
   H2.Headers.of_list
     [ ("te", "trailers"); ("content-type", "application/grpc+proto") ]
 
-let call ~service ~rpc ?(scheme = "https") ~handler ~do_request
+let call ~service ~rpc ?(scheme = "https") ~handler ~(do_request : do_request)
```

Comment:
Do we really need this type annotation?
```suggestion
let call ~service ~rpc ?(scheme = "https") ~handler ~do_request
```

Reply:
Yes, else we need the even more verbose `?trailers_handler:(Some trailers_handler)` instead of just `~trailers_handler` in the call later. But maybe you have another idea?

Reply:
Of course I could also put the type annotation later in the call, if you prefer that style.

Reply:
Right. Sorry, my bad. I don't have strong opinions about this, and I don't have another idea.